dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/ArtifactDownloader.java

```diff
@@ -75,11 +106,13 @@ public void synthesizeAndDownloadFeatures(List<String> mavenCoords, String dLoca
         }
         updateProgress(progressBar.getMethodIncrement("establishConnection"));
         info(Messages.INSTALL_KERNEL_MESSAGES.getLogMessage("STATE_MAVEN_REPO_CONNECTION_SUCCESSFUL"));
+
         try {
             missingFeatures = ArtifactDownloaderUtils.getMissingFiles(featureURLs, envMap);
-        } catch (IOException e) {
+        } catch (IOException | InterruptedException | ExecutionException e) {
```

Comment:
what code changed here that requires us to catch these additional exceptions?

Reply:
It's from the ExectuorService that gets thrown from ArtifactDownloaderUtils.getMissingFiles(featureURLs, envMap);